1.9/usage/faq.md

```diff
@@ -32,7 +32,7 @@ A comprehensive overview of a few common service discovery implementations is av
 
 ## Is it possible to span my cluster over different cloud providers?
 
-The current implementation of DC/OS only supports a Single Availability Zone/Failure Zone per Cluster and one single region per cluster.
+Yes. For more information, see [this document](/docs/1.9/administration/installing/high-availability-setup/).
```

Comment:
This is not technically correct. Having one cluster span multiple cloud providers is "hybrid cloud" which we do not currently support.